common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -278,7 +278,7 @@ public double getDouble(String key) {
     throw new RuntimeException("Invalid configuration key " + key + ".");
   }
 
-  private float getFloat(String key, final float defaultValue) {
+  private float getFloat(String key) {
```

Comment:
thanks for the catch, yes it should be `public`.
and yes, it is not really used by other functions yet. But I feel this util method might be good to have.